Homework_05/task_01.py

```diff
@@ -0,0 +1,20 @@
+# 1. Create a program that will receive a number and print the corresponding number in the Fibonacci list using
+# generators
+print("### Fibonacci Generator ###")
+while True:
+    def get_fibonacci(number):   # create generator
+        fib1 = fib2 = 1
+        for i in range(number):
+            yield fib1
+            fib1, fib2 = fib2, fib1 + fib2
+
+
+    try:
+        input_number = int(input("-> "))
+
+        generator_fibonacci = get_fibonacci(input_number)
+        fibonacci_list = list(generator_fibonacci)
+
+        print(f"{input_number} number in the Fibonacci sequence: {fibonacci_list[-1]}\n")
+    except:
```

Comment:
Because of this construction, it is impossible to exit the program with keyboard interruption. Maybe you should add more specific error handling here or add some way to exit the program